### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by UnrealApex**

### CODE REVIEW

**Comment by Miodec** [comment]
Made a little update, let me know what you think

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,18 +76,38 @@
...
+
+### Theme Guidelines
+<!-- TODO: add screenshots to provide examples for dos and don'ts -->
+- Make sure your theme is unique and a simmilar looking one is not already avaiable
+- The text color is either black or white (or very close to)
+- Your theme has been added to the `_list` file and the `textColor` property is the theme's main color
+- Your theme is clear and readable with both `flip test colors` and `colorful mode` enabled and disbled
+
+### Language Guidelines
+- Do not include swear words
```
Comment (UnrealApex) [inline CONTRIBUTING.md:100]: or words that would not be considered appropriate in a workplace environment

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -48,18 +76,38 @@
...
+- Ensure that your contribution meets JSON standards (no trailing comma at the end of a list)
+- Be sure to add your language to the `_list` and `_groups` files
+- Make sure the number of words corresponds to the file name (for example: `languageName.json` is 200 words, `langugeName_1k.json` is 1000 words, and so on)
 
-1. Code style is enforced by [Prettier](https://prettier.io/docs/en/install.html), which is automatically ran every time you `git commit` (if you've followed the above instructions properly).
-1. If you are unsure of the best practices and conventions of Javascript, please take a look at the [JavaScript Best Practices](https://www.w3schools.com/js/js_best_practices.asp) and [JavaScript Coding Conventions](https://www.w3schools.com/js/js_conventions.asp)
+### Quote Guidelines
+- Do not include swear words
```
Comment (UnrealApex) [inline CONTRIBUTING.md:106]: Please refrain from adding quotes that contain swear words or censor them.

**Comment by UnrealApex** [comment]
> Made a little update, let me know what you think

Looks good, added some comments detailing some small revisions.

File: CONTRIBUTING.md (hunk 1 above)
Comment (Miodec) [reply CONTRIBUTING.md:105]: Thats pretty much the same though isnt it

**Review by Miodec — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (UnrealApex) [reply CONTRIBUTING.md:105]: Technically yes.

**Pull request merged**